compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2615,6 +2615,18 @@ impl TargetOptions {
             }
         })
     }
+
+    pub(crate) fn has_neg_feature(&self, search_feature: &str) -> bool {
+        self.features.split(',').any(|f| {
+            if let Some(f) = f.strip_prefix('-')
+                && f == search_feature
+            {
+                true
+            } else {
+                false
+            }
+        })
```

Comment:
this would be correct also, right?
```suggestion
            f.strip_prefix('-').map(|f| f == search_feature).unwrap_or(false)
```

Reply:
I guess it does? I don't find `map`+`unwrap_or` very easy to read.

But I think I can use `is_some_and` here.